index.js

```diff
@@ -201,18 +203,21 @@ var animationQueue = new AnimationQueue();
 // Give each notification a unique id
 var latestID = 0;
 
-function notify(title, text, url, image, onClickFunc, onShowFunc, onCloseFunc) {
+function notify(title, text, url, image, sound, onClickFunc, onShowFunc, onCloseFunc) {
```

Comment:
What do you mean by dynamic args? ´Object` (it currently does accept that)? `arguments` parsing?